cookietemple/create/github_support.py

```diff
@@ -207,8 +207,7 @@ def prompt_github_repo(dot_cookietemple: Optional[dict]) -> Tuple[bool, bool, bo
 
     # No dot_cookietemple_dict was passed -> prompt whether to create a Github repository and the required settings
     create_git_repo, private, is_github_org, github_org = False, False, False, ''
-    print('[bold blue]You will now be asked whether to create a GitHub repository with cookietemple for your project.\n Note that this'
-          'is highly recommended; otherwise you will not be able to use some of cookietemples features!\n')
+    print('[bold blue]Creating a GitHub repo is highly recommended; otherwise you will not be able to use some of cookietemples features!\n')
```

Comment:
Not happy yet.
Maybe
"Automatically creating a Github repository with cookietemple is strongly recommended. Otherwise you will not be able to use all of cookietemple's features."